src/Tools/Database/keros.sql

```diff
@@ -575,4 +575,34 @@ INSERT INTO treso_facture_type (id, label) VALUES
   (3, 'intermediaire'),
   (4, 'solde');
 
+DROP TABLE IF EXISTS treso_payment_slip;
+CREATE TABLE treso_payment_slip (
+    id int(11) NOT NULL AUTO_INCREMENT,
+    missionRecapNumber varchar(32),
+    consultantName varchar(255),
+    consultantSocialSecurityNumber varchar(255),
+    addressId int(11),
+    email varchar(255),
+    studyId int(11) NOT NULL,
+    clientName varchar(255),
+    projectLead varchar(255),
+    isTotalJeh boolean,
+    isStudyPaid boolean,
+    amountDescription varchar(2048),
+    createdDate date,
```

Comment:
Je pense qu'un certain nombre d'attributs ici devraient être not null, mais je ne sais pas quelles info sont indispensables à un paiement slip. Je vous laisse voir ! (je pense notamment à creatorId, createdDate, isStudyPaid, clientName et le reste à voir en détail)

Reply:
Aussi createdDate c'est un Datetime ou Date ? Dans le fichier de l'entité c'était Datetime

Reply:
Sont en `not null` les attributs obligatoires de la doc. Ici il n'y a que l'id. On pourrait réfléchir à changer ça (mais une autre fois je pense du coup).
Corrigé pour le `datetime`. J'ai déplacé la création de la table, elle était à la fin du fichier